document/src/main/java/com/ritense/document/domain/impl/searchfield/SearchFieldDataType.java

```diff
@@ -27,26 +27,26 @@ public enum SearchFieldDatatype {
 
     @JsonValue private final String name;
 
-    SearchFieldDatatype(String name) {
+    SearchFieldDataType(String name) {
         this.name = name;
     }
 
-    public static SearchFieldDatatype fromString(String text) {
-        for (SearchFieldDatatype datatype : SearchFieldDatatype.values()) {
-            if (datatype.name.equalsIgnoreCase(text)) {
-                return datatype;
+    public static SearchFieldDataType fromString(String text) {
+        for (SearchFieldDataType dataType : SearchFieldDataType.values()) {
+            if (dataType.name.equalsIgnoreCase(text)) {
+                return dataType;
             }
         }
-        throw new IllegalStateException(String.format("Cannot create SearchFieldDatatype from string %s", text));
+        throw new IllegalStateException(String.format("Cannot create SearchFieldDataType from string %s", text));
     }
 
-    public static SearchFieldDatatype fromKey(String key) {
-        for (SearchFieldDatatype datatype : SearchFieldDatatype.values()) {
-            if (datatype.name().equalsIgnoreCase(key)) {
-                return datatype;
+    public static SearchFieldDataType fromKey(String key) {
+        for (SearchFieldDataType dataType : SearchFieldDataType.values()) {
```

Comment:
This seems like a direct copy of fromString. Why not call fromString(..) from here?

Reply:
All `.fromKey(...)` methods weren't used. So I removed them